web/src/utils/dashboard/convertDataIntoUnitValue.ts

```
@@ -281,43 +283,75 @@
-      return value * 60 * 60 * 1000;
-    case "d": // Days
-      return value * 24 * 60 * 60 * 1000;
-    case "w": // Weeks
-      return value * 7 * 24 * 60 * 60 * 1000;
-    case "M": // Months (approximate, using 30.44 days per month)
-      return value * 30.44 * 24 * 60 * 60 * 1000;
-    default:
-      return 0; // Return 0 if the unit is not recognized
-  }
-}
+    if (isNaN(periodValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)